VideoPrototype/VideoPrototype/Main/Controller/Model Controllers/CameraController.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  CameraController.swift
+//  VideoPrototype
+//
+//  Created by Kenny on 6/3/20.
+//  Copyright © 2020 Hazy Studios. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+protocol CameraUIDelegate: AVCaptureFileOutputRecordingDelegate {
+    func playMovie(url: URL)
+    var cameraView: CameraPreviewView! { get set }
+}
+
+class CameraController {
```

Comment:
Great job on making your code modular and adhering to MVC and Moving the captureCameraSession config to a sperate custom class, I'm noticing that you're making certain methods private due to the fact that no other object needs to interact with them, which is good meanwhile the ones that arent private are the ones that need to be called inside a View Controller. Great job on implementing a protocol.